CMakeLists.txt

```diff
@@ -225,6 +226,10 @@ elseif(CORE_SYSTEM_NAME STREQUAL rbpi)
   core_require_dep(MMAL)
 endif()
 
+if(PYTHON_FOUND)
```

Comment:
yes i know, but i found it a bit tedious (and possibly confusing) to introduce a layer on top of the built-in python find module just to set the definition. but if that's what you want..